es6.md

```diff
@@ -89,3 +89,52 @@ interface ArrayPattern <: Pattern {
     elements: [ Pattern | null ];
 }
 ```
+
+# Classes
+
+```js
+interface Class <: Node {
+    id: Identifier | null;
+    superClass: Expression;
+    body: ClassBody;
+}
+```
+
+## ClassBody
+
+```js
+interface ClassBody <: Node {
+    type: "ClassBody";
+    body: [ MethodDefinition ];
+}
+```
+
+## MethodDefinition
+
+```js
+interface MethodDefinition <: Node {
+    type: "MethodDefinition";
+    key: Identifier;
+    value: FunctionExpression;
+    kind: "" | "get" | "set";
```

Comment:
Not really the same - class method definition is something that can appear and be used only on this class and instances, while in object expression it's just shorthand syntax for regular `Property` with `FunctionExpression` that can be replaced with any other `value` later.